lms/services/lti_user.py

```diff
@@ -0,0 +1,56 @@
+from lms.models.lti_user import LTIUser, display_name
+from lms.services import LTIRoleService
+
+
+class LTIUserService:
```

Comment:
This service could also contain this:

https://github.com/hypothesis/lms/blob/09c3da9cc73c2a030f093f2719567e0a80438d70/lms/security.py#L225


as `from_request` or similar.


I'd save that for a follow up PR if we go with the service approach.